website/docs/how-to/how-to-enable-openapi.mdx

```diff
@@ -79,7 +79,7 @@ docker run \
 
 The configuration option for enabling OpenAPI and the swagger UI is `enableOAS`. Set this option to `true`.
 
-The following examples have been shortened to show the only the relevant configuration options. For more detailed instructions on how to run Unleash or the proxy, refer to [how to run the Unleash proxy](how-to-run-the-unleash-proxy.mdx) or the [section on running Unleash via Node.js from the deployment section](/using-unleash/deploy/getting-started.md#option-three---from-nodejs) of the documentation.
+The following examples have been shortened to show only the relevant configuration options. For more detailed instructions on how to run Unleash or the proxy, refer to [how to run the Unleash proxy](how-to-run-the-unleash-proxy.mdx) or the [section on running Unleash via Node.js from the deployment section](/using-unleash/deploy/getting-started.md#option-three---from-nodejs) of the documentation.
```

Comment:
Should we mention edge here?

Reply:
Probably :) I'll create an issue for it. thanks!